src/patternfly/components/Table/styles.scss

```diff
@@ -0,0 +1,792 @@
+    cursor: pointer;
+    background-color: transparent;
+    border: none;
+
+  }
+  /* stylelint-enable */
+
+  // First child padding right
+  &:first-child button {
+    padding-left: var(--pf-c-table-cell--first-child--PaddingLeft);
+  }
+
+  // Last child padding right
+  &:last-child button {
+    padding-right: var(--pf-c-table-cell--last-child--PaddingRight);
+  }
+}
+
+// Sort indicator
+.pf-c-table__sort-indicator {
+  margin-left: var(--pf-c-table__sort-indicator--MarginLeft);
+
+  // set line-height to 1 so it does not adjust height of button
+  line-height: 1;
+  color: var(--pf-c-table__sort-indicator--Color);
+  pointer-events: none;
+}
+
+
+// Compact table
+// ==================================================================
+
+.pf-c-table.pf-m-compact {
+  thead #{$pf-c-table-standard-th} {
+    padding:
+      var(--pf-c-table--m-compact-th--PaddingTop)
+      var(--pf-c-table--m-compact--PaddingRight)
+      var(--pf-c-table--m-compact-th--PaddingBottom)
+      var(--pf-c-table--m-compact--PaddingLeft);
+  }
+
+  tbody {
+    #{$pf-c-table-standard-td},
+    #{$pf-c-table-standard-th} {
+      padding:
+        var(--pf-c-table--m-compact--PaddingTop)
+        var(--pf-c-table--m-compact--PaddingRight)
+        var(--pf-c-table--m-compact--PaddingBottom)
+        var(--pf-c-table--m-compact--PaddingLeft);
+      font-size: var(--pf-c-table--m-compact--FontSize);
+    }
+  }
+
+  // First row has 2x padding as subsequent rows
+  tbody tr:first-child {
+    #{$pf-c-table-standard-th},
+    #{$pf-c-table-standard-td} {
+      padding-top: calc(var(--pf-c-table--m-compact--PaddingTop) * 2);
+    }
+
+    .pf-c-table__toggle,
+    .pf-c-table__check,
+    .pf-c-table__action {
+      padding-top: var(--pf-c-table--m-compact--PaddingTop);
+    }
+  }
+
+  // Last row has 2x padding as subsequent rows
+  tbody tr:last-child {
+    #{$pf-c-table-standard-th},
+    #{$pf-c-table-standard-td} {
+      padding-bottom: calc(var(--pf-c-table--m-compact--PaddingBottom) * 3);
+    }
+
+    .pf-c-table__toggle,
+    .pf-c-table__check,
+    .pf-c-table__action {
+      padding-bottom: calc(var(--pf-c-table--m-compact--PaddingBottom) * 2);
+    }
+  }
+
+  tbody #{$pf-c-table-standard-row} {
+    border: none;
+  }
+
+  .pf-c-table__toggle,
+  .pf-c-table__check,
+  .pf-c-table__action {
+    padding-top: 0;
+    padding-bottom: 0;
+    vertical-align: middle;
+  }
+
+  .pf-c-table__check {
+    padding-right: var(--pf-c-table--m-compact--PaddingRight);
+  }
+}
+
+// Icon inline
+// replace this when inline text with icon is in place
+.pf-c-table__icon-inline {
+  display: flex;
+  align-items: center;
+
+  > *:not(:last-child) {
+    margin-right: var(--pf-c-table__icon-inline--MarginRight);
+  }
+}
+
+// Modifier - Wrap
+.pf-c-table .pf-m-wrap {
+  white-space: normal;
+}
+
+// Modifier - Width
+/* stylelint-disable */
+.pf-c-table [class*="pf-m-width-"] {
+  table-layout: fixed;
+}
+
+.pf-c-table .pf-m-width- {
+  &10 {
```

Comment:
this breaks the rules :) I get why you're doing it, but it makes the code less searchable.